--- a/scene/singleton/world.gd
+++ b/scene/singleton/world.gd
@@ -141,7 +141,19 @@
 
 
 
+func CreateLobby() -> void :
+	await _standard_down()
+	var lobby: Control = load("res://net/lobby.gd").new()
+	primary_capsule.add_child(lobby)
+	await _standard_up()
+
+
+func _unhandled_input(event: InputEvent) -> void :
+	if event is InputEventKey and event.pressed and not event.echo and event.keycode == KEY_F9:
+		CreateLobby()
+
+
 func _ready() -> void :
 	if M.data.player.first_time and not M.THISISDEMO:
 		CreateTutorial()
